.env

```diff
@@ -9,3 +9,9 @@ MAILGUN_API=fhlakjshdf8932pidkjshfasdjfy89234ks83pikanzkjahslka
 JWT_PRIVATE_KEY_LOCATION=config/keys/jwtRS256.key
 JWT_PUBLIC_KEY_LOCATION=config/keys/jwtRS256.key.pub.pem
 JWT_SECRET_KEY=123
+
+KEYCLOAK_CONFIGS_URI=https://brokencrystals/keycloak/auth/realms/master/.well-known/openid-configuration
+KEYCLOAK_USERS_URI=https://brokencrystals/keycloak/auth/admin/realms/master/users
+KEYCLOAK_USERS_NAME=admin
+KEYCLOAK_USERS_PASSWORD=Pa55w0rd
```

Comment:
As I mentioned before, you should use `client_credential` grant type to get access to `keycloak`

See https://www.appsdeveloperblog.com/keycloak-rest-api-create-a-new-user/

Reply:
You said that we can use `grant_type: 'password'` for simplicity

Reply:
😆 you've messed up again. You should always use `client_credentials` when we speak about S2S interaction.

Reply:
I suggest you to do a meeting notes. We discussed it several times (including daily meetings) with same result:
"Set a grant type to password for simplicity" as you said

Reply:
Yeah, that what I does (and you should, too). I talked about that in term of C2S and not S2S interaction.